.gitignore

```diff
@@ -1,7 +1,133 @@
+# Perl unit test
```

Comment:
I have just followed the same idea as for `ensembl-py`. Excluding is always "easier" in my opinion, so you see quickly when you are picking something you shouldn't. But will wait for the second reviewer to make a decision.

Reply:
It's a totally subjective thing, in both cases you could see quickly when you are not picking up something you should or picking up something you shouldn't! I suggest a git include rather than ignore, because that is less likely to change so much or be added to, and in this particular case, the anti-ignore list would be smaller than the to be ignored list. I am happy either way, but wanted to put forward the suggestion for consideration :nerd_face:

Reply:
```suggestion
# Ignore everything
*
# Except these dirs
!conf/
!docs/
!modules/
!pipelines/
!scripts/
!sql/
!src/
!travisci/
!xs/
# And these files
!.codecov.yml
!cpanfile
!DEPRECATED.md
!.gitignore
!LICENSE
!.mypy.ini
!NOTICE
!perlcriticrc
!PIP_VERSION
!pull_request_template.md
!pylintrc
!README.md
!run_all_tests.sh
!setup.py
!.travis.yml
!yamllintrc
```